src/Options/index.js

```diff
@@ -181,6 +181,12 @@ export interface ParseServerOptions {
   Requires option `verifyUserEmails: true`.
   :DEFAULT: false */
   emailVerifyTokenReuseIfValid: ?boolean;
+  /* Set to `false` to prevent sending of verification email. Supports a function with a return value of `true` or `false` for condiitonal email sending.
+  <br><br>
+  Default is `true`.
+  <br>
+  :DEFAULT: true */
+  sendUserEmailVerification: ?boolean;
```

Comment:
If this supports also a function, should the type reflect that?

Reply:
I look it up and it seems not to be possible to define a generic function type for a param. Not sure how that's done in JSDoc, or what the custom definitions scripts are expecting. If the `void` you've added does the trick, then I think we can close this conversation.